manual/googlechrome-extensions/googlechrome-emberinspector/googlechrome-ember-inspector.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-ember-inspector</id>
+    <version>1.0.0</version>
+    <title>Ember Inspector</title>
+    <authors>Ember Development Team</authors>
+    <owners>gep13 chocolatey</owners>
```

Comment:
Owners should be a comma separated list as per [.nuspec Reference](https://docs.nuget.org/ndocs/schema/nuspec)